client/my-sites/media-library/list-no-content.jsx

```diff
@@ -74,7 +74,7 @@ class MediaLibraryListNoContent extends Component {
 	render() {
 		let line = '';
 		let action = '';
-		const showPexelsButton =
+		const showFreeLibraryButton =
```

Comment:
Should this button link to Openverse now?

Reply:
I'm not sure, let me look at that in a follow-up PR!